### PR TITLE
feat: aether 2.0.0-rc

### PR DESCRIPTION
**Pull request by lordmallam**

### CODE REVIEW

File: _base_/docker-compose.yml [hunk 1]
```diff
@@ -40,3 +40,15 @@ services:
...
+  redis:
+    image: redis:alpine
+    command: >
+      redis-server
+      --requirepass ${REDIS_PASSWORD}
+      --notify-keyspace-events KEA
+      --appendonly yes
+      --auto-aof-rewrite-percentage 100
+      --auto-aof-rewrite-min-size 64mb
+    networks:
```
Comment (obdulia-losantos) [inline _base_/docker-compose.yml]: Could you add this and check that everything works? (Google disables a couple of commands by default)

```yml
    environment:
      - DISABLE_COMMANDS=FLUSHDB,FLUSHALL
```

File: tests/docker-compose.yml [hunk 2]
```diff
@@ -16,6 +16,18 @@ services:
     networks:
       - test
 
+  redis-test:
+    image: redis:alpine
```
Comment (obdulia-losantos) [inline tests/docker-compose.yml]: redis or ardb? @shawnsarwar

**Comment by shawnsarwar** [comment]
Is this still current?

**Comment by obdulia-losantos** [comment]
Waiting for release 2.0 launch...

**Comment by obdulia-losantos** [comment]
Split into #153 and #154

**Pull request closed without merging**